### DISCUSSION TITLE
Cannot find sbcore.ini in Slingplayer 2.0.4.521

### DISCUSSION THREAD

**Discussion by shoobydoodoo** [GerryDazoo/Slinger#574] [Q&A] [answered]
Hi all,

I'm trying to check the key codes for my remote. I'm using a Pro-HD. I followed Gerry's release note V3.08 and under the section "Tips and Tricks," it says to find sbcore.ini in C:\ProgramData\Sling Media\Slingplayer\2.0. I just downloaded Slingplayer 2.0.4.521 [from here](https://softfamous.com/slingplayer/) and I can stream fine over LAN, but I cannot find this 2.0 folder and sbcore.ini. Here is what my Slingplayer folder [looks like](https://imgur.com/a/XwqopHR). Is this Slingpalyer not the correct 2.0.4 version? If so, could someone with the correct install file please share with me? Thanks so much for any help.

**Comment by mike-uk** [comment, marked as answer]
@shoobydoodoo I don't know if this is the right one for you but on the JP1 remote forum there is another one. You need to register to download. The download button is at an odd place between the 2 borders very easy to miss but its appear after you login.
http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=97

**Reply by shoobydoodoo** [reply]
Thanks Mike, 

I tried ver 2.0.0.465 in that JP1 remote forum and it doesn't have the file directory instructed in the release notes either. Like the 2.0.4, there is a SPConfig.ini and a SBCore.sdb file under C:\Program Files (x86)\Sling Media\SlingPlayer, but no sbcore.ini. When I opened the SPConfig.ini, it does have the [main] title, but when I tried to add the line "DebugRemote=1", it would not let me save, saying I'm not administrator and have no permission to make changes. [Here are the screenshots.](https://imgur.com/a/gJ3pdBt) I wonder why the discrepancy between what Gerry says in the release note and what I'm seeing. Perhaps @GerryDazoo and @f520k510 could give us some hints?

**Reply by f520k510** [reply]
It should be in c:\program data\sling media\slingplayer/2,0/

**Reply by shoobydoodoo** [reply]
ahh! I found it. I realized I didn't turn on the "Hidden Item" view in Windows view. Thank you guys so much!!
